brave/brave-core
# Element Picker on Android

Resolves https://github.com/brave/brave-browser/issues/33241

- Enabled Element picker for Android
- Enhanced Element Picker's UI 
- Added Feature flag which is enabled by default for Desktop and disabled by default for Android

https://github.com/user-attachments/assets/f8a0bfcf-0af0-4f40-8c41-de6e59152ad4





## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. Open Android Brave browser
2. Open any webpage you want to block any HTML element
3. Open `Brave Shields menu` -> `Advanced controls`
4. Make sure that `Block elements` item is absent (It means that `brave://flags/#block-element-feature` flag is disabled)
5. If it was disabled, (It was not enabled with Griffin), enable it and make sure that `Block elements` item is present in the `Brave Shields menu` -> `Advanced controls` menu
6. Click `Block elements` and make sure that `Element Picker` appears
7. Click any HTML element you want to block and press button `Block Element`, and make sure element blocked
8. Change Android Theme (Dark/Light) and make sure that `Element picker` has the same color
